yarn.lock

```diff
@@ -9878,7 +9878,7 @@ cypress-real-events@^1.2.0:
   resolved "https://registry.yarnpkg.com/cypress-real-events/-/cypress-real-events-1.3.0.tgz#d737d68dd590b28c71756d63fbf36b863e936912"
   integrity sha512-IYkhC1C9tGR7eN5d4VmT/28swyYeRmj+c+e0YcblnnbF68CVAtpc4D+v1JiENZA4il8W5XEcI/FjI64ss2lIag==
 
-cypress@^6.6.0:
+cypress@6.6.0, cypress@^6.6.0:
```

Comment:
Will make a follow up PR into v0 for single version policy, currently uses `^6.6.0`

Reply:
actually seems that syncpack will break, I don't want to add an exception so doin g it now